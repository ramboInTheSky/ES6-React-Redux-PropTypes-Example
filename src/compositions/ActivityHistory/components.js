import styled from 'styled-components';
import { Button, Typography } from 'nhh-styles';

export const Activity = styled.div`
  margin-bottom: 1rem;
`;

export const ActivityBody = styled.div``;

export const ActivityLabel = styled.span`
  font-weight: 700;
`;

export const ActivityLink = styled(Button)`
  font-size: inherit;
`;

export const Wrapper = styled.div`
  clear: both;
  margin-bottom: 1.5rem;
  padding-top: 1rem;
`;

export const H3 = styled(Typography.H3)`
  margin-bottom: 30px;
`;

export const ActivityDescription = styled.span`
  word-break: break-word;
`;
