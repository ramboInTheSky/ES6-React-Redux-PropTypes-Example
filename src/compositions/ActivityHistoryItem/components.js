import styled from 'styled-components';
import { Box, BoxHighlight, mediaQueries, defaultTheme } from 'nhh-styles';

export const ActivityItemWrapper = styled.div`
  margin: 30px -50px;
  min-height: 18.75rem;
  ${mediaQueries.mobile`
    margin: 0;
  `};
`;

export const CustomerDetailsContainer = styled(BoxHighlight)`
  margin: 0 0 30px 0;
`;

export const LoaderContainer = styled.div`
  min-height: 18.75rem;
  display: flex;
  flex-direction: column;
  justify-content: center;
`;

export const Wrapper = styled(Box)`
  padding: 25px;
  margin-bottom: 30px;
`;

export const AttachmentsWrapper = styled.div`
  margin-bottom: 30px;
`;

export const ErrorMessage = styled.div`
  margin-bottom: 30px;
  color: ${defaultTheme.colors.support.two};
`;

export const TableWrapper = styled.div`
  table {
    font-size: 0.8em;
    td:nth-child(2) {
      text-align: right;
      button,
      button:hover {
        color: ${defaultTheme.colors.primary} !important;

        &.success {
          color: ${defaultTheme.colors.support.three} !important;
        }
      }
    }
  }
`;
