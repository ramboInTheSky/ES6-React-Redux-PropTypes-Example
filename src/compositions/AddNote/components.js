import styled from 'styled-components';
import { Box, BoxHighlight, Button } from 'nhh-styles';

export const FormContainer = styled.div`
  margin: 0 50px;
`;

export const Link = styled(Button)`
  font-weight: 300;
  font-size: 18px;
`;

export const Wrapper = styled(Box)`
  padding: 25px;
  margin-bottom: 30px;
`;

export const CustomerDetailsContainer = styled(BoxHighlight)`
  margin: 0 0 30px 0;
`;
