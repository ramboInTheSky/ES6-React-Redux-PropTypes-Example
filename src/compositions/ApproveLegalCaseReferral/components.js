import styled from 'styled-components';
import { Button } from 'nhh-styles';

// eslint-disable-next-line
export const Link = styled(Button)`
  font-size: 18px;
  padding: 8px 0 0 0;
  text-align: left;
  display: block;
`;
