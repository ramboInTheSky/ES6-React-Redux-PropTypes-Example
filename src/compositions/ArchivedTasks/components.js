import styled from 'styled-components';
import { Box } from 'nhh-styles';

export const Wrapper = styled(Box)`
  padding: 25px;
  margin-bottom: 30px;
`;

export const LinkedItems = styled.div`
  text-align: center;
  margin: 0 0 30px 0;
`;
