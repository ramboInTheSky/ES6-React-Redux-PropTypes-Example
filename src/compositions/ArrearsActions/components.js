import styled from 'styled-components';

export const Wrapper = styled.div`
  margin-top: 1.875rem;
`;

export const MenuItemsWrapper = styled.div`
  padding: 1.25rem;
`;

export const ItemWrapper = styled.div`
  margin-bottom: 0.625rem;
  &:last-of-type {
    margin-bottom: 0;
  }
`;
