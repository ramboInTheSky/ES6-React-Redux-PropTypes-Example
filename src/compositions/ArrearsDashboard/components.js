import styled from 'styled-components';

// eslint-disable-next-line import/prefer-default-export
export const PatchesWrapper = styled.div`
  margin-bottom: 10px;
`;
