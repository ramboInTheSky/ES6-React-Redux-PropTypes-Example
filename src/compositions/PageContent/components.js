import styled from 'styled-components';

// eslint-disable-next-line
export const Content = styled.div``;
