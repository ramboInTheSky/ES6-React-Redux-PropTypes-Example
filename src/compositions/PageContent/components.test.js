import React from 'react';
import { shallow } from 'enzyme';

import 'jest-styled-components';

import { Content } from './components';

describe('Page components', () => {
  it('should render <Content /> correctly', () => {
    expect(shallow(<Content />)).toMatchSnapshot();
  });
});
