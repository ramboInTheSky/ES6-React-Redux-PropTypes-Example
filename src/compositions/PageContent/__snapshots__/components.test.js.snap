// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page components should render <Content /> correctly 1`] = `
<div
  className=""
/>
`;
